code/_onclick/item_attack.dm

```diff
@@ -14,6 +14,16 @@
 
 /mob/living/attackby(obj/item/I, mob/user, params)
 	user.changeNext_move(CLICK_CD_MELEE)
+
+	if((butcher_results) && (stat == DEAD))
```

Comment:
cats, foxes..goats....monkeys..things with a butcher result..i end up attacking them..not butchering them.